### PR TITLE
Added code of conduct in Readme.md

### LINKED ISSUE #1346: "Code of Conduct" and "Reporting Guidelines"
I was reading the conda-forge docs, In the Governance section I found the code of conduct, but what I feel is we must link the code of conduct or can write them in brief and add in the readme.md file.  Any contributor who would be visiting the page will be reading the readme.md file first and they shd be well aware of the code of conduct and reporting guidelines, and the consequences incase that is violated.

Not sure if this can be implemented, but I would like to know ur views on this

### PR DESCRIPTION
**Pull request by chicken-biryani**
This issue closes #1346 

PR Checklist:

- [ ] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [X] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [ ] put any other relevant information below

### CODE REVIEW

**Comment by chicken-biryani** [comment]
Hi, @beckermr and @kathatherine could you please review this PR and let me know if there are any required changes? Thanks! :)

**Comment by chicken-biryani** [comment]
@beckermr on it!

**Comment by ngam** [comment]
Thank you so much for this!!

I would like to encourage you to add a very brief statement about the consequence of violating our CoC. I believe there is something about that in the docs; basically, repeated violations will lead to a ban.

**Comment by chicken-biryani** [comment]
Hi, @ngam that does make it more friendly and welcoming in nature! Thanks! :)

@ngam and @beckermr could you let me know if I could better the current PR?

**Comment by chicken-biryani** [comment]
@ngam I came across the Reporting guidelines i.e https://numfocus.org/code-of-conduct#reporting-guidelines
and 'Enforcement: What Happens After a Report is Filed?' i.e https://numfocus.org/code-of-conduct#enforcement

Shall I add both? Or do you think the Reporting guidelines are enough?

**Comment by beckermr** [comment]
Please do not bump all of core on these PRs.

**Comment by chicken-biryani** [comment]
Sorry, @beckermr I didn't mean to.

I believe it is auto-mated like over here https://github.com/conda-forge/conda-forge.github.io/pull/1471 and here- https://github.com/conda-forge/conda-forge.github.io/pull/1620

**Comment by beckermr** [comment]
ah no worries!

**Comment by ngam** [comment]
> Please do not bump all of core on these PRs.

> I believe it is auto-mated like over here #1471 and here- #1620

Yes, it is automated because of this line:
https://github.com/conda-forge/conda-forge.github.io/blob/5132c8d1d9e8886d9f778517b6a8350bbfe063c2/.github/CODEOWNERS#L1

**Comment by ngam** [comment]
> Shall I add both? Or do you think the Reporting guidelines are enough?

Yes, please, that's a good idea!

**Comment by chicken-biryani** [comment]
Added the above.

Shall I make Reporting guidelines as sub-headings instead? Or is this okay?

**Comment by ngam** [comment]
I think this looks good! It makes it more prominent if they have their own subheading. Maybe @beckermr has more thoughts?

File: README.md [hunk 1]
```diff
@@ -28,6 +28,28 @@ If you have questions or need help, please check out our documentation for a [li
...
+
+## Reporting Guidelines
+
+If you believe someone is violating the code of conduct, please report this in a timely manner. Code of conduct violations reduce the value of the community for everyone. The team at conda-forge takes reports of misconduct very seriously and is committed to preserving and maintaining the welcoming nature of our community.
+
+All reports will be kept confidential. Please have a look at the [Reporting Guidelines](https://numfocus.org/code-of-conduct#reporting-guidelines).
+
+## Enforcement: What Happens After a Report is Filed?
+
+conda-forge and/or our event staff will attempt to ensure your safety and help with any immediate needs, particularly at an in-person event. NumFOCUS will make every effort to acknowledge receipt within 24 hours (and we’ll aim for much more quickly than that). Have a look at the process of [What Happens After a Report is Filed?](https://numfocus.org/code-of-conduct#enforcement).
```
Comment (beckermr) [inline README.md:51]: We cannot speak for numfocus here.

Comment (chicken-biryani) [reply README.md]: Ah! My bad! Fixed it!

File: README.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ All reports will be kept confidential. Please have a look at the [Reporting Guid
 
 ## Enforcement: What Happens After a Report is Filed?
 
-conda-forge and/or our event staff will attempt to ensure your safety and help with any immediate needs, particularly at an in-person event. NumFOCUS will make every effort to acknowledge receipt within 24 hours (and we’ll aim for much more quickly than that). Have a look at the process of [What Happens After a Report is Filed?](https://numfocus.org/code-of-conduct#enforcement).
+conda-forge and/or our event staff will attempt to ensure your safety and help with any immediate needs, particularly at an in-person event. conda-forge will make every effort to acknowledge receipt within 24 hours (and we’ll aim for much more quickly than that). Have a look at the process of [What Happens After a Report is Filed?](https://numfocus.org/code-of-conduct#enforcement).
```
Comment (beckermr) [inline README.md:51]: These reports do not go to conda-forge directly, so this section needs to be reworded a bit more.

**Comment by chicken-biryani** [comment]
Added a small fix as well. Although it isn't concerned with the targeted issue. Fixed the link of dev meeting minutes

File: README.md (hunk 2 above)
Comment (chicken-biryani) [reply README.md:51]: Will do so.

I am a little concerned though and may have messed up a little. I have copied and pasted the wordings of NumFocus directly into the sections. Shall I modify the wordings of other sections as well or let it be?

**Comment by chicken-biryani** [comment]
@beckermr updated as suggested!

File: README.md [hunk 3]
```diff
@@ -28,9 +28,31 @@ If you have questions or need help, please check out our documentation for a [li
 
 **Note: "All changes must be made in the `/src` folder and NOT in the `/docs` folder. Html files in the ./docs folder are auto generated "**
 
+## Code of conduct
+
+We at conda-forge adhere to the [NumFOCUS Code of Conduct](https://numfocus.org/code-of-conduct).
+
+Be kind to others. Do not insult or put down others. Behave professionally. Remember that harassment and sexist, racist, or exclusionary jokes are not appropriate for conda-forge.
+
+All communication should be appropriate for a professional audience including people of many different backgrounds. Sexual language and imagery is not appropriate.
```
Comment (kathatherine) [inline README.md:37]: A comma should be placed between "audience" and "including".

File: README.md [hunk 4]
```diff
@@ -28,9 +28,31 @@ If you have questions or need help, please check out our documentation for a [li
...
+
+## Reporting Guidelines
+
+If you believe someone is violating the code of conduct, please report this in a timely manner. Code of conduct violations reduce the value of the community for everyone. The team at conda-forge takes reports of misconduct very seriously and is committed to preserving and maintaining the welcoming nature of our community.
+
+All reports will be kept confidential. Please have a look at the [Reporting Guidelines](https://numfocus.org/code-of-conduct#reporting-guidelines).
+
+## Enforcement: What Happens After a Report is Filed?
+
+conda-forge's team and/or our event staff will try to ensure your safety and help with any immediate needs, particularly at an in-person event. Once we have received the report through the relevant authorities, conda-forge will make every effort to acknowledge the receipt and take action. (and we’ll aim for much more quickly than that). Have a look at the process of [What Happens After a Report is Filed?](https://numfocus.org/code-of-conduct#enforcement).
```
Comment (kathatherine) [inline README.md:51]: There is no need for the period after "action" since the words in the parenthesis are also part of the sentence.

**Review by kathatherine — CHANGES_REQUESTED** [review]
Looks great except for those two comments.

File: README.md (hunk 4 above)
Comment (chicken-biryani) [reply README.md]: Fixed!

**Comment by kathatherine** [comment]
@chicken-biryani You should be fine quoting NumFocus's code of conduct directly, I've learned. They actually have a template others can follow: https://github.com/numfocus/numfocus/blob/master/manual/numfocus-coc.md

Your additions look good to me.

File: README.md [hunk 5]
```diff
@@ -26,6 +26,30 @@ If you have questions or need help, please check out our documentation for a [li
 4.  Make and commit your changes.
 5.  Submit a [pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests) to the main repository proposing your changes.
 
+**Note: "All changes must be made in the `/src` folder and NOT in the `/docs` folder. Html files in the ./docs folder are auto generated "**
+
```
Comment (jaimergp) [inline README.md:30]:
```suggestion
```

**Review by jaimergp — APPROVED** [review]
LGTM.

Will merge next week unless @conda-forge/core has any further comments.

**Review by ocefpaf — APPROVED** [review]

**Comment by ocefpaf** [comment]
> Will merge next week unless @conda-forge/core has any further comments.

This one slipped through the cracks. Let's merge!

**Pull request merged**